shared/chat/conversation/index.desktop.js

```diff
@@ -97,14 +97,12 @@ class Conversation extends Component<void, Props, State> {
         <Banner />
```

Comment:
one of the big goals is to have less churn when properties change. so we pass less stuff down so less stuff rerenders